Cargo.lock

```diff
@@ -53,7 +53,7 @@ version = "0.1.0"
 dependencies = [
  "anyhow",
  "chrono",
- "clap 3.1.3",
```

Comment:
There are many changes in the lock file which wasn't mentioned in the PR description. I think you might have accidentally run `cargo update`. It would be cleaner to not update the lock file in this PR. You could reset it with `git restore --source origin/main -- Cargo.lock`.

Reply:
Sorry, this was not my intention. Thanks. I removed it.